nf_core/modules/lint.py

```diff
@@ -519,6 +519,7 @@ def lint_module_tests(self):
         # Lint the test.yml file
         try:
             with open(self.test_yml, "r") as fh:
+                # TODO: verify that the tags are correct
```

Comment:
Shall we leave this for another PR?

Reply:
yeah I would say, nothing I want to do too quickly